proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+## I. Overview
+This governance policy describes how an open source project can formally join the Foundation via the [Project Proposal Process](). It describes the [Stages]() a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the [Annual Review Process]() through which those changes will be evaluated and made. 
+
+Benefits of the project progression process include:
+* Promotion and public visibility of work that can add value to the _$FOUNDATION_ mission and build the ingredients of a successful project
+* Deepened alignment with existing projects
+* Mentorship and support 
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to _$FOUNDATION_ legal, code of conduct and IP Policy requirements
+
+This proposal has been modified from the [CNCF process documentation](https://github.com/cncf/toc/tree/master/process).
+
+## II. Project Proposal Process
+
+### Introduction
+This governance policy sets forth the proposal process for projects to be accepted into the _$FOUNDATION_. The process is the same for both existing projects which seek to move into the _$FOUNDATION_, and new projects to be formed within the _$FOUNDATION_.
+
+### Project Proposal Requirements
+Projects must be proposed via GitHub. Project proposals submitted to the _$FOUNDATION_ must provide the following information to the best of their ability:
+
+* name of project
+* project description (what it does, why it is valuable, origin and history)
+* statement on alignment with _$FOUNDATION_ charter mission
+* sponsor from CPC (sponsor helps mentor projects)
+* preferred maturity level 
+* license 
+* source control (GitHub by default)
+* external dependencies (including licenses)
```

Comment:
For npm modules this is _hard_, as the dependency tree is huge.

Reply:
There are tools to gather this information, and JSF projects all have access to [FOSSA](https://fossa.io)... but I _do_ want to know why this is required.

Even better: for each of these items, I'd like to publish a rationale of why it's important and necessary (even if that is mainly legalese).

Reply:
cynically speaking, I'm guessing this is so companies can avoid projects using (GPL) non-permissive (GPL) OSS (GPL) licenses, and/or non-OSI-approved licenses